File: ZMusic-Plugin/src/main/java/me/zhenxin/zmusic/event/EventBukkit.java
package me.zhenxin.zmusic.event;

import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerJoinEvent;

public class EventBukkit implements Listener {

    @EventHandler
    public void onPlayerJoinEvent(PlayerJoinEvent event) {
        Event.onJoin(event.getPlayer());
    }
}

File: src/main/kotlin/me/zhenxin/zmusic/event/EventBukkit.kt
package me.zhenxin.zmusic.event

import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerJoinEvent


class EventBukkit : Listener {
    @EventHandler
    fun onPlayerJoinEvent(event: PlayerJoinEvent) {
        EventEx.onJoin(event.player)
    }
}
